src/routes/Selection/index.tsx

```diff
@@ -7,18 +7,22 @@ import {
   getSelection,
   deleteSelection,
   checkPopulationRegistraion,
+  sortSelection,
 } from "../../store/selection/actions";
+import { SortOptions } from "../../store/selection/types";
 
 interface SelectionContainerProps extends State {
   getSelection: typeof getSelection;
   deleteSelection: typeof deleteSelection;
+  sortSelection: typeof sortSelection;
   checkPopulationRegistraion: typeof checkPopulationRegistraion;
 }
 
 const SelectionContainer: React.FC<SelectionContainerProps> = ({
   selection,
   getSelection,
   deleteSelection,
+  sortSelection,
```

Comment:
This is just passed down to the Selection component, you could just import it in the Selection file.